logging/README.md

```diff
@@ -57,32 +57,31 @@ command `git clone <https_url>`.
 
 ### Customize the Deployment
 
-### USER_DIR
+The process of customizing the logging deployment consists of: 
+- creating the location for your local customization files
+- using the `USER_DIR` environment variable to specify the location of the customization files
+- specifying customization variables and parameters in the customization files
 
-Setting the `USER_DIR` environment variable allows for any user customizations
-to be stored outside of the directory structure of this repository. The default
-`USER_DIR` is the root of this repository. A directory referenced by `USER_DIR`
-should include `user*` files in the same relative structure as they exist in
-this repository.
+See the [main README](../README.md) to for information about the customization process.
```

Comment:
You can link directly to the customization section with **../README.md#customization** as the link.